packages/mobile/src/verify/VerificationEducationScreen.test.tsx

```diff
@@ -2,11 +2,22 @@ import * as React from 'react'
 import 'react-native'
 import { render } from 'react-native-testing-library'
 import { Provider } from 'react-redux'
+import { features } from 'src/flags'
 import { Screens } from 'src/navigator/Screens'
 import VerificationEducationScreen from 'src/verify/VerificationEducationScreen'
 import { createMockStore, getMockStackScreenProps } from 'test/utils'
 
 describe('VerificationEducationScreen', () => {
+  const komenciEnabled = features.KOMENCI
+
+  beforeAll(() => {
+    features.KOMENCI = true
+  })
+
+  afterAll(() => {
+    features.KOMENCI = komenciEnabled
+  })
+
   it('shows the `skip` button when already verified', () => {
```

Comment:
nit: maybe we want to test the KOMENCI=false case too?

Reply:
While doing this I discovered something that was missing:

If the user had 0 balance, we were still showing the `Insufficient balance` button instead of the `Continue` button since a balance is not necessary for Komenci.